unit_tests/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+include_directories("${gtest_SOURCE_DIR}/include")
+
+enable_testing()
+
+ADD_SUBDIRECTORY(noddy)
```

Comment:
Boringly: Can we the noddy directory and Holder.cpp so that we don't have a Noddy Holder reference please?

Reply:
Sorry I don't understand, there seems to be a word missing

Reply:
I think the missing word is probably *rename* ... We might have to keep the Noddy Holder joke private to ourselves...!

Reply:
Renamed to Noddy Test